test/integration/helpers/extensions.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+// to use: a template literal is probably the easiest way to provide JS to inject into the
+// extensions.js file.  Please wrap in IIFE as addExtension() may be called multiple times &
+// the file contents will be concatonated:
+//    var jsToWrite = `
+//    (function() {
+//      // createFromURL test file ${testTime} // (can use interpolation))
+//      window.OPENSHIFT_CONSTANTS.CREATE_FROM_URL_WHITELIST = ['openshift', 'template-dumpster'];
+//    })();
+//    `;
+// adding to the extension.js file:
+//    addExtension(jsToWrite)
+exports.addExtension = function(fileContents) {
```

Comment:
That whole directory gets blown away every time we build.  running the tests over and over will generate all new files.  Still think its necessary?